.github/workflows/test-coverage.yml

```
@@ -0,0 +1,13 @@
+name: 'coverage'
+on: push
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v1
```

Should we also add names for these steps?
```suggestion
      - name: Checkout
        uses: actions/checkout@v1
```

---

✅ 👌 80a0b46